src/Backend/Modules/Pages/Domain/Page/Form/PageDataRedirectType.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Backend\Modules\Pages\Domain\Page\Form;
+
+use Backend\Modules\Pages\Engine\Model as BackendPagesModel;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\UrlType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class PageDataRedirectType extends AbstractType implements PageDataTypeInterface
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add(
+            'redirect',
+            ChoiceType::class,
+            [
+                'label' => 'lbl.Redirect',
+                'choices' => [
+                    'lbl.None' => 'none',
+                    'lbl.InternalLink' => 'internal',
+                    'lbl.ExternalLink' => 'external',
+                ],
+                'label_attr' => ['class' => 'custom-control-label radio-custom'],
```

Comment:
Please expand this array. Having both single line and expanded arrays in the same method causes readability issues

Reply:
fixed in 6e3ace50b